learn/get-started/build-a-data-service-in-ballerina.md

```diff
@@ -471,15 +481,16 @@ bal run
 You view the output below.
 
 >**Info:** This creates an `/employees` endpoint on port `8080`, which can 
-be accessed via a browser by visiting `http://locahost:8080/employees`.
+be accessed via a browser by visiting `http://localhost:8080/employees`.
```

Comment:
```suggestion
be accessed by visiting `http://localhost:8080/employees`.
```

Reply:
Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)